erpnext/public/js/utils/serial_no_batch_selector.js

```diff
@@ -261,7 +261,8 @@ erpnext.SerialNoBatchSelector = class SerialNoBatchSelector {
 
 			return frappe.db.get_list("Serial No", {
 				filters: { 'name': ["in", selected_serial_nos]},
-				fields: ["batch_no", "name"]
+				fields: ["batch_no", "name"],
+				limit: 999
```

Comment:
@hereabdulla 999 looks like a large limit for a UI component, is this actually needed? Can you share a gif of how more than 100 serial no looks like in the selector?

Reply:
but it is taking only 20 nos while fetching but we might have needed more than that